content/notes/concepts.md

```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
 
 
 * **Access**: optimizing for social welfare will mean selecting a business model which prioritizes high levels of access. By contrast, business models which use closed rights frameworks may enable you to make more money (although this is not always the case, as businesses such as Facebook demonstrate). 
-* **Quality:** optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. This is a matter of cost-benefit analysis, trading off gains in quality most often against levels of provision (i.e. the same resources can get you more lower quality digital goods or fewer higher quality digital goods) By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
+* **Quality**: optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
```

Comment:
Maybe, for fairness add a comment lkie ...

Obviously, business models that generate little revenue will  struggle to maintain quality.